### ISSUE TITLE
`Communication:` Not saving the draft of a message

### ISSUE THREAD

**Issue by JanaNF** [ls1intum/Artemis#9771]
### Describe the bug

When I write a message but do not send the message, I leave the channel and then rejoin the channel, and my message gets deleted. I want it to be saved like a draft.

### To Reproduce

1. Go to a (private) text channel
2. Write a message but do not send it
3. Leave the channel and go to another one
4. Rejoin the channel
5. Your draft was deleted

### Expected behavior

See my saved message in the message field.

### Which version of Artemis are you seeing the problem on?

7.6.5

### What browsers are you seeing the problem on?

Chrome

**Comment by jonas-de** [comment]
I want to follow up on this, this would be very useful. My main use case is to gather additional information from lectures or other chats when writing a message.

For me, it would be enough to store the draft in local/session storage.

**Comment by krusche** [comment]
I agree this would be a nice feature, we try to implement this in the webapp and also consider to implement it in the mobile apps (@eylulnc)

**Comment by Kiara65** [comment]
This was implemented in https://github.com/ls1intum/Artemis/pull/10944
